### PR TITLE
EDU-588 Support chapter Q and A in display mode

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-588

Screen Shot 2022-06-08 at 18 29 59

Screen Shot 2022-06-08 at 18 30 11

Screen Shot 2022-06-08 at 18 31 39

### CODE REVIEW

File: src/plugins/interactive-media/interactive-media.yml [hunk 1]
```diff
@@ -40,3 +40,12 @@ addAnswer:
 markCorrectAnswer:
   en: Mark as correct answer
   de: Als richtige Antwort markieren
+continue:
+  en: continue
+  de: fortfahren
+replay:
+  en: replay
+  de: wieder spielen
```
Comment (ahelmberger) [inline src/plugins/interactive-media/interactive-media.yml:48]: "noch einmal abspielen" or "wiederholen" would sound better. Also, I think we have all button labels start with a capital letter (in both languages), so these three labels have a little bit of an "unfamiliar look".

File: src/styles/mixins.less [hunk 2]
```diff
@@ -161,3 +161,9 @@
   margin-left: auto;
   margin-right: auto;
 }
+
+.m-overflow-ellipsis {
```
Comment (ahelmberger) [inline src/styles/mixins.less:165]: Nice refactoring :+1:

File: src/plugins/interactive-media/interactive-media.less [hunk 3]
```diff
@@ -115,18 +113,71 @@
...
 
   &.is-active {
-    color: @edu-green-icon-color;
+    color: @edu-success-color;
   }
 }
 
 .InteractiveMediaDisplay-overlay {
   position: absolute;
+  padding: 15px;
```
Comment (ahelmberger) [inline src/plugins/interactive-media/interactive-media.less:122]: Or maybe the equivalent `@edu-content-padding`?

File: src/styles/mixins.less (hunk 2 above)
Comment (adrianaluchian) [reply src/styles/mixins.less:165]: 🙇

File: src/plugins/interactive-media/interactive-media.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/plugins/interactive-media/interactive-media.yml]: True, changed to "Wiederholen" and capitalized first letter in both languages, thanks!

File: src/plugins/interactive-media/interactive-media.less (hunk 3 above)
Comment (adrianaluchian) [reply src/plugins/interactive-media/interactive-media.less]: Changed!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**